pages/post/_slug/edit/_id.vue

```diff
@@ -25,6 +25,9 @@ export default {
       if (!this.Post || !this.Post[0].id) {
         return
       }
+      if (this.Post[0].author.id !== this.$store.getters['auth/user'].id) {
```

Comment:
This is very dirty. Isn't there a VueJS plugin for authorization?

Reply:
No but nuxt middleware. But currently I don’t know if it would work with vue-Apollo